### PR TITLE
Update version strings in Gallery

### LINKED ISSUE #9259: Gallery "about" screen still says 2016
We can update that to Q2 2017 :)

### PR DESCRIPTION
**Pull request by jason-simmons**
Fixes https://github.com/flutter/flutter/issues/9259

### CODE REVIEW

**Comment by jason-simmons** [comment]
@sethladd

File: examples/flutter_gallery/lib/gallery/drawer.dart [hunk 1]
```diff
@@ -209,9 +209,9 @@ class GalleryDrawer extends StatelessWidget {
 
     final Widget aboutItem = new AboutListTile(
       icon: const FlutterLogo(),
-      applicationVersion: '2016 Q3 Preview',
+      applicationVersion: 'April 2017 Preview',
       applicationIcon: const FlutterLogo(),
-      applicationLegalese: '© 2016 The Chromium Authors',
+      applicationLegalese: '© 2016-7 The Chromium Authors',
```
Comment (Hixie) [inline examples/flutter_gallery/lib/gallery/drawer.dart]: Just make this 2017.

**Comment by Hixie** [comment]
LGTM

File: examples/flutter_gallery/lib/gallery/drawer.dart (hunk 1 above)
Comment (jason-simmons) [inline examples/flutter_gallery/lib/gallery/drawer.dart]: done

**Pull request merged**